arch/arm/boot/dts/overlays/README

```diff
@@ -874,9 +874,14 @@ Info:   Initiates a shutdown when GPIO pin changes. The given GPIO pin
         can be powered up again by driving GPIO3 low. The default
         configuration uses GPIO3 with a pullup, so if you connect a
         button between GPIO3 and GND (pin 5 and 6 on the 40-pin header),
-        you get a shutdown and power-up button.
+        you get a shutdown and power-up button. Please note that older
+        Raspberry PI 1 rev 1 uses GPIO1 instead of GPIO3.
```

Comment:
We prefer to write it "Raspberry Pi", but the device in question was the "Model B rev 1" (which is long enough on its own without the "Raspberry Pi" at the start), so please replace all occurrences of "Raspberry PI 1" with "Model B".